app/src/androidTest/java/org/mozilla/focus/activity/screenshots/ScreenGrabTest.java

```diff
@@ -95,6 +95,8 @@ protected void beforeActivityLaunched() {
                         .setBody(TestHelper.readTestAsset("rabbit.jpg")));
                 webServer.enqueue(new MockResponse()
                         .setBody(TestHelper.readTestAsset("download.jpg")));
```

Comment:
Below is needed to load the site when opening a new tab in line 368. (not necessary, but cleaner)